### PR TITLE
Merge Final Commits for FacMan

### PR DESCRIPTION
**Pull request by AlexanderKidd**
-Minor changes to error handling responses
-General refactoring, such as parameter order in functions
-Changed some DB types (from a month ago)

### CODE REVIEW

**Comment by ZacharyMoran** [comment]
Why in the world did you wait for 67 commits to push this.

File: Database/DummyData.sql [hunk 1]
```diff
@@ -91,11 +91,20 @@ INSERT INTO Section (MAX_STUDENTS, PROFESSOR_ID, COURSE_ID, TERM_ID, CLASSROOM_I
 
 INSERT INTO Student_Section (STUDENT_ID, SECTION_ID) VALUES (1, 1);
 INSERT INTO Student_Section (STUDENT_ID, SECTION_ID) VALUES (2, 1);
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (ZacharyMoran) [inline Database/DummyData.sql]: You have left multiple Merge Conflicts in this PR

File: Database/DummyData.sql [hunk 2]
```diff
@@ -91,11 +91,20 @@ INSERT INTO Section (MAX_STUDENTS, PROFESSOR_ID, COURSE_ID, TERM_ID, CLASSROOM_I
...
+=======
+=======
+>>>>>>> origin/master
```
Comment (ZacharyMoran) [inline Database/DummyData.sql]: Merge Conflict

File: Database/DummyData.sql (hunk 1 above)
Comment (roda0313) [inline Database/DummyData.sql]: Are you aware that you literally commited a merge problem? This PR is such a mess that i think im going to close it and copy the changes over and resubmit a PR

File: Database/DummyData.sql [hunk 3]
```diff
@@ -91,11 +91,20 @@ INSERT INTO Section (MAX_STUDENTS, PROFESSOR_ID, COURSE_ID, TERM_ID, CLASSROOM_I
...
 INSERT INTO Student_Section (STUDENT_ID, SECTION_ID) VALUES (2, 3);
 INSERT INTO Student_Section (STUDENT_ID, SECTION_ID) VALUES (2, 2);
 INSERT INTO Student_Section (STUDENT_ID, SECTION_ID) VALUES (5, 4);
 INSERT INTO Student_Section (STUDENT_ID, SECTION_ID) VALUES (3, 1);
 INSERT INTO Student_Section (STUDENT_ID, SECTION_ID) VALUES (4, 6);
+<<<<<<< HEAD
+>>>>>>> refs/remotes/roda0313/master
+=======
+>>>>>>> origin/master
```
Comment (ZacharyMoran) [inline Database/DummyData.sql]: Merge Conflict

File: Database/TeamHumanResources.sql [hunk 4]
```diff
@@ -7,9 +7,8 @@ CREATE TABLE UniversityEmployee(
 	MANAGER_ID INTEGER NOT NULL,
 	TITLE TEXT NOT NULL,
 	ADDRESS TEXT NOT NULL,
-    PHONE TEXT NOT NULL,
```
Comment (ZacharyMoran) [inline Database/TeamHumanResources.sql]: Don't Change Other Team SQL

**Pull request closed without merging**